### PR TITLE
Ch/feature/footer

### LINKED ISSUE #2: Navigation Components
# Story 
As a user, I want to go to the homepage so that I can see a navigation bar with the title of the site and home button. 

As a normal sighted user, I will see a navigation bar that tells me the site name of "Face It" and have a home button. When I click on the home button, nothing happens. 

### Concerns 
- [x] Apply Route to Home
- [x] Redirect applied to Home
- [x] Do we want our logo here, too?

### LINKED ISSUE #3: Category Component
# Story 
As a user, I want to go to the homepage so that I can see the available categories to sort through. 

As a normal sighted user, when I go to the homepage, I will see three icons & associated titles telling me what the categories are. If I try to click into one, nothing happens. 

### Concerns 
- [x] Are the sizes relative to screen size (mobile first)
- [x] What icons will be used
- [x] Will each have it's own accent color?

### LINKED ISSUE #5: Mission Statement
# Story 
As a user, I want to go to the homepage so that I can see an obvious and clearly written mission statement of the site. 

As a sighted user, I see a clearly defined mission statement that doesn't exceed more than a 1/4 of my screen on mobile. 

### Concerns 
- [x] What's important to us? 
- [x] Do we want each category to have it's own mission statement/explanation as to what that category means?

### LINKED ISSUE #7: Express Back-up Server
# Story 
As a dev, I want to create a back up server using express so that I don't have to worry about the original API crashing/being limited to api rate limits . 

When I create a fetch call to the our back up server, it functions as if I had made a fetch call to the original makeup APi

### Concerns 
- [x] Do we need to worry about POSTs at all?

### LINKED ISSUE #14: Router to Categories
# Story 
As a dev, I want to implement Router so that I can use url paths rather than conditional rendering to move users through our web app. 

If a user is able to use the back and forward buttons and navigate between two pages and data is still displayed as expected. If a user is able to navigate to a page with a url only and images are displayed as expected. 

### Concerns 
- [x] Route & redirects as needed?

### PR DESCRIPTION
**Pull request by conconartist**
- [x] Feature
- [ ] Refactor
- [ ] Bug Fix 
- [ ] Clean Up

Sets up initial layout and foundation for the app, with React, Router, and Cypress.  The App component was modified to render with Router and links to categories are clickable but do not have data yet displayed.  The title and icon show with a navigation bar at the top.  The three categories are shown and are responsive according to pixel width of the window.  

The server for the Makeup API we are using was causing issues before we realized that it is set up to reach a limit of calls that we were exceeding. We proceeded to download the data to post on a backup server (in another repo).

> Describe your code changes in detail for reviewers.

File folders within the Components folder of the project are updated with Component functionality we anticipate adding in future iterations.  The static comp of the main display is complete in the App component.  The icon component "Logo" is an addition as well.  

All code is new aside from the starter code from the Create React app.

In your terminal, enter `npm start` once you are in the project directory to open the server.  You should see the static page in your browser (localhost: 3000).

#### Checklist:
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] My code follows the style guidelines of this project
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/App/App.css [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  font-family: 'Raleway',
+  sans-serif;
+}
+
+h2 {
+  border: solid 1px #D1925C;
+  font-size: 1.25rem;
+  display: flex;
+  justify-content: center; 
+  padding: 2rem;;
```
Comment (gaj23) [inline src/components/App/App.css:15]: There's an extra semicolon to be deleted.

**Review by kelsiebesingeryeh — APPROVED** [review]
Looks great!

File: src/components/App/App.css [hunk 2]
```diff
@@ -0,0 +1,84 @@
+@import url('https://fonts.googleapis.com/css2?family=Raleway&display=swap');
+
+body {
+  background: #F4E9E1;
+  color: #1F0800;
+  font-family: 'Raleway',
+  sans-serif;
```
Comment (gaj23) [inline src/components/App/App.css:7]: This is best put all on one line

File: src/components/App/App.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+    this.state = {
+      makeup: [],
+      vegan: [],
+      fairTrade: [],
+      eco: []
+    }
+  }
+
+  // componentDidMount(){
+  //   fetch('http://makeup-api.herokuapp.com/api/v1/products.json')
```
Comment (gaj23) [inline src/components/App/App.js:20]: I'm proud of the solution we decided on and will implement due to the low rate limit of our chosen api.

File: src/index.css [hunk 4]
```diff
@@ -0,0 +1,13 @@
+body {
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
+    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+}
+
+code {
```
Comment (gaj23) [inline src/index.css:10]: I think we can delete this?

**Pull request merged**